### PR TITLE
Allow for non-javax look and feel class name

### PR DESCRIPTION
**Pull request by YoungJules**
Couldn't get it to build on my Linux system.
My laf is com.sun.java.swing.plaf.gtk.GTKLookAndFeel

### CODE REVIEW

File: src/test/java/com/itextpdf/rups/RupsConfigurationTest.java [hunk 1]
```diff
@@ -128,7 +128,9 @@ public void setSystemLookAndFeel() {
         RupsConfiguration.INSTANCE.setLookAndFeel(laf);
         RupsConfiguration.INSTANCE.saveConfiguration();
         String lookAndFeel = RupsConfiguration.INSTANCE.getLookAndFeel();
-        Assertions.assertTrue(lookAndFeel.contains("javax.swing.plaf") || lookAndFeel.equals("com.sun.java.swing.plaf.windows.WindowsLookAndFeel"));
+        Assertions.assertTrue(lookAndFeel.contains("javax.swing.plaf")
+            || lookAndFeel.contains("java.swing.plaf")
```
Comment (YoungJules) [inline src/test/java/com/itextpdf/rups/RupsConfigurationTest.java:132]: Hmmm, looking at it again, I guess we don't need the WinBlows-specific check as it'll pass my broader check just for java.swing.plaf anyway

**Review by RonSkons — APPROVED** [review]
Looks good to me. I agree that the second check is not strictly necessary, but the purpose of the test is clearer with it.

**Comment by RonSkons** [comment]
Hi @YoungJules , could you please move your code to a named branch?
e.g. `improvement/Linux_LaF_Fix` or similar.
Jenkins doesn't like that the branch on your fork is also named `develop`.

**Comment by RonSkons** [comment]
Fixed as part of #104

**Pull request closed without merging**